projects/plugins/jetpack/extensions/blocks/contact-form/editor.scss

```diff
@@ -95,6 +95,22 @@
 	}
 }
 
+.jetpack-contact-form.is-placeholder {
```

Comment:
Since the `BlockVariationPicker` was holding the placeholder border I had to remove it and apply the border and some spacing to the wrapping element.
The color `#1e1e1e` used was just copied from the `BlockVariationPicker` styles.